### DISCUSSION TITLE
Bb-qq/r8152 package on DS1520+

### DISCUSSION THREAD

**Discussion by brentb636** [community/community#26718] [Actions]
<p>Trying to report that the <a href="https://github.com/bb-qq/r8152" class="inline-onebox" rel="noopener nofollow ugc">GitHub - bb-qq/r8152: Synology DSM driver for Realtek RTL8152/RTL8153/RTL8156 based adapters</a> package works on the DS1520+, using the Gemini Lake package driver.  I couldn’t figure how to notify the success of the driver on the link page.</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>it’s best to raise an issue at the repo itself,</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/bb-qq/r8152/issues">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/bb-qq/r8152/issues" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/47092464/181099366-d1635802-72f8-4e2e-817b-71fa0ae52906.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/3/1303ba88d41e8e584aa3889789f5f6a1f599c24b_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/1/3/1303ba88d41e8e584aa3889789f5f6a1f599c24b_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/1/3/1303ba88d41e8e584aa3889789f5f6a1f599c24b.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/3/1303ba88d41e8e584aa3889789f5f6a1f599c24b_2_10x10.png"></div>

<h3><a href="https://github.com/bb-qq/r8152/issues" target="_blank" rel="noopener">Issues · bb-qq/r8152</a></h3>

  <p>Synology DSM driver for Realtek RTL8152/RTL8153/RTL8156 based adapters - Issues · bb-qq/r8152</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>they are the best ones to handle it</p>
